.github/workflows/reuseaction.yml

```
@@ -0,0 +1,19 @@
+#                   
+# This source file is part of the Apodini open source project
+#
+# SPDX-FileCopyrightText: 2019-2021 Paul Schmiedmayer and the Apodini project authors (see CONTRIBUTORS.md) <paul.schmiedmayer@tum.de>
+#
+# SPDX-License-Identifier: MIT
+#  
+
+name: REUSE Compliance Check
+
+on: [push, pull_request]
```

I would suggest to add the same triggers as for the SwiftLint action:
```suggestion
on:
  pull_request:
  workflow_dispatch:
```